src/fontra/views/editor/panel-reference-font.js

```diff
@@ -59,7 +61,44 @@ function cleanFontItems(fontItems) {
     return {
       uplodadedFileName: fontItem.uplodadedFileName,
       fontIdentifier: fontItem.fontIdentifier,
+      supportedLanguages: fontItem.supportedLanguages,
```

Comment:
In the context of my other feedback, this line should be removed:
```suggestion
```

Reply:
I did it because cleanFontItems is called when a font item is deleted in the panel.
I will delete supportedLanguages from fontItem. It's complicating the process. I will probably cache them in panel instance.

Reply:
Ah I see, I think I misunderstood what you were doing here. Let's keep this field in the "clean" font item. I mixed it up with adding the *current* supported languages to the model.

Reply:
I deleted it, it confuses the reader including me after looking at the code a couple of days later. I store the supported languages in panel instance.